app/src/main/java/org/mozilla/fenix/components/toolbar/DefaultToolbarMenu.kt

```diff
@@ -14,6 +14,7 @@ import mozilla.components.browser.menu.BrowserMenuBuilder
 import mozilla.components.browser.menu.BrowserMenuHighlight
```

Comment:
(unrelated to this LoC, just putting it here so we can chain comments)

> You need to call `invalidate` on the toolbar menu so it checks the list again (not a great API right now...sorry!). There's a helper class that just calls invalidate in certain conditions.

@NotWoods hmm, I don't know if we can.  We don't have a reference to the view here, and it's a required parameter for invalidate.  I remember going through this problem with Christian a few months ago and being more or less stumped.  Do you have any suggestions?

Reply:
You could override `onReaderableStateUpdated` in `MenuPresenter`

Reply:
Works great, thanks for the help!